integrated_channels/utils.py

```diff
@@ -407,17 +408,17 @@ def batch_by_pk(ModelClass, extra_filter=Q(), batch_size=10000):
         qs = ModelClass.objects.filter(pk__gt=start_pk).filter(extra_filter).order_by('pk')[:batch_size]
 
 
-def truncate_item_collections(items_to_create, items_to_update, items_to_delete, combined_maximum_size):
+def truncate_item_dicts(items_to_create, items_to_update, items_to_delete, combined_maximum_size):
```

Comment:
because we're truncating later in the workflow, we are now working with `dict` data structures instead of `list` - this truncate needed to be reworked and retested